### ISSUE TITLE
How can I use this on my project, not on my shareable config?

### ISSUE THREAD

**Issue by domenic** [sarbbottam/eslint-find-rules#69]
My project has a non-shareable config. I was hoping for a workflow where I install this globally and do

```
$ eslint-find-rules .eslintrc.json
```

and it shows me which rules I haven't configured. However, this failed in two ways:

- First, it wouldn't work until I also globally installed eslint. That's sad.
- Second, even after I did that, it always output nothing. I tried deleting random rules from my config, and nothing happened.

**Comment by sarbbottam** [comment]
>
```
$ eslint-find-rules .eslintrc.json
```

You have to specify an `option`. For example, if you want to list the unused rules, use the `-u` option.

```
$ eslint-find-rules -u .eslintrc.json
```

Please refer the [usage section](https://github.com/sarbbottam/eslint-find-rules#usage) for details.


> * First, it wouldn't work until I also globally installed eslint. That's sad.

I wonder how come it did not throw any error, [it excepts `eslint` to be installed locally](https://github.com/sarbbottam/eslint-find-rules/blob/master/src/lib/rule-finder.js#L55)

> * Second, even after I did that, it always output nothing. I tried deleting random rules from my config, and nothing happened.

Could you try, installing `eslint` locally and specifying an `option`; `eslint-find-rules -u .eslintrc.json`?

**Comment by domenic** [comment]
I already have eslint installed locally; it threw an error until I also installed it globally.

It looks like specifying an option is what I needed; thank you! I skimmed the readme and found "By default it will error out only for -u|--unused" which I misread as "by default it will specify -u|--unused"
